apps/authentication/const.py

```diff
@@ -31,6 +31,7 @@ def get_prop_backends(cls, confirm_type):
 class MFAType(TextChoices):
     OTP = 'otp', _('OTP')
     SMS = 'sms', _('SMS')
+    Email = 'email', _('Email')
     Face = 'face', _('Face Recognition')
     Radius = 'otp_radius', _('Radius')
     Custom = 'mfa_custom', _('Custom')
```

Comment:
There is an unnecessary semicolon at the end of line 31 which might lead to syntax errors depending on where this code is used. Additionally, there seems to be confusion regarding what exactly "MFAType" represents based on provided details such as using TextChoices from typing module. To simplify it, it should just use choices object directly if you need a list instead of enum. There's also minor style change needed but it won't cause issue unless it becomes part of larger project structure.